internal/controllers/core/tiltfile/api.go

```diff
@@ -557,11 +557,19 @@ func toClusterObjects(nn types.NamespacedName, tlr *tiltfile.TiltfileLoadResult,
 		return result
 	}
 
+	var annotations map[string]string
+	if tlr.FeatureFlags[feature.ClusterRefresh] {
+		annotations = map[string]string{
+			"features.tilt.dev/cluster-refresh": "true",
+		}
+	}
```

Comment:
This felt like a reasonably clean way to propagate feature flag information to an individual resource, and there's sorta precedent here because we conditionally create disable buttons in a sibling function based on feature flags